### DISCUSSION TITLE
Why does Polygon Edge have continuous blocks created ?

### DISCUSSION THREAD

**Discussion by spunkyshooter** [0xPolygon/polygon-edge#630] [Q&A] [answered]
As said [here](https://github.com/0xPolygon/polygon-edge/issues/443#issuecomment-1063360114) Polygon-Edge is creating new blocks every 2 sec, why ?

We have deployed Polygon-edge on production and we are now trying to integrate blockscout for explorer and since these blocks would be visible to users, its concerning us.

![image](https://user-images.githubusercontent.com/29997888/177923041-6ff8298f-115b-402a-8c72-41550394ac6c.png)

**Comment by zivkovicmilos** [comment, marked as answer]
Hey @spunkyshooter,

Thank you for reaching out.

Essentially, Polygon Edge operates on the notion that empty block are perfectly normal.
That means your consensus network is in tact and working to produce (albeit empty) blocks.

Currently, there is no support to enable any kind of non-empty block production mode (if there are no transactions to put into a block, don't produce a block), however we are looking into having this sometime down the road. Would you be interested in this feature?

What's worrying you about this exactly?

**Reply by spunkyshooter** [reply]
Thanks @zivkovicmilos 

Yes, we are interested in that feature, From the users POV, they expect to see the non-empty blocks and not continuous empty blocks, They think this is a testnet. This small point could raise the questions and trustability of the blockchain.

**Reply by ConejoCapital** [reply]
any updates on this? incredibly curious on how it makes a difference for explorer UX and DB costs

**Reply by Solidityghost998** [reply]
any updates on the same ??

**Reply by ZIDUK** [reply]
Any update?

**Reply by kalpeshkamble** [reply]
??

**Comment by rhesus-protocol** [comment]
Yes, I was also thinking about the same. The miner address is always zero address even if I am running 2 validator nodes, why the miner is always zero address?

**Reply by Kourin1996** [reply]
@rhesus-protocol Hello, `Miner` in Block doesn't point to block creator in IBFT consensus. This field is used for voting to change validator set in IBFT PoA consensus.

**Reply by rhesus-protocol** [reply]
Hi, thanks for you reply. I am right now using PoS consensus. Is it same for PoS too?

**Reply by Kourin1996** [reply]
Yes. Miner field is not set.
